README.md

```
@@ -0,0 +1,121 @@
+# Softeer Bootcamp 5th DE course
+
+### Guideline
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)